cppapi/server/tango_config.h

```diff
@@ -81,11 +81,15 @@
             #define WIN32_VC9
         #elif ((_MSC_VER >= 1600) && (_MSC_VER < 1700))
             #define WIN32_VC10
-		#elif ((_MSC_VER >= 1700) && (_MSC_VER < 1800))
-			#define WIN32_VC11
-		#elif (_MSC_VER >= 1800)
-			#define WIN32_VC12
-        #endif   // VC8+/VC9/VC10/VC11/VC12
+        #elif ((_MSC_VER >= 1700) && (_MSC_VER < 1800))
+            #define WIN32_VC11
+        #elif ((_MSC_VER >= 1800) && (_MSC_VER < 1900))
+            #define WIN32_VC12
+        #elif ((_MSC_VER >= 1900) && (_MSC_VER < 1910))
+            #define WIN32_VC14
+        #elif (_MSC_VER >= 1910)
+            #define WIN32_VC141
```

Comment:
It looks like there is no added value in adding a special case for VC141 for the moment so I would keep it simple for the moment and define WIN32_VC14 if (_MSC_VER >= 1900).